### PR TITLE
fix Roaring64Bitmap toArray bug

### PR DESCRIPTION
**Pull request by caoxiaoyuan7242**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version = 0.9.3-SNAPSHOT
+version = 0.9.4-SNAPSHOT
```
Comment (weijietong) [inline gradle.properties:1]: don't change the version number at one PR

File: RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java [hunk 2]
```diff
@@ -777,6 +783,37 @@ public void testAnd() {
     assertEquals(1, left.getLongCardinality());
     assertEquals(123, left.select(0));
   }
+  @Test
+  void testToArrayAfterAnd(){
+      Random random = new Random();
```
Comment (weijietong) [inline RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java:788]: I think this test case was not designed well to reproduce the case that the result `Roaring64Bitmap` has empty container after some `and` operations.Please redesign it.

File: gradle.properties (hunk 1 above)
Comment (caoxiaoyuan7242) [reply gradle.properties:1]: ok，

**Review by weijietong — COMMENTED** [review]
@caoxiaoyuan7242 The fix looks great ,  please solve my left comments, tks.

File: RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java (hunk 2 above)
Comment (caoxiaoyuan7242) [reply RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java:788]: This test case just want to  test toArray function is correct after bitmap use and operation.I think empty container after and operations has no problem.Root cause is getLongIterator function was not considered this.

Comment (caoxiaoyuan7242) [reply RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java:788]: I see what you mean. I'll modify the test case

File: RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java [hunk 3]
```diff
@@ -25,11 +25,14 @@
 import java.util.List;
 import java.util.Random;
 import java.util.Set;
+
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.roaringbitmap.RoaringBitmap;
 import org.roaringbitmap.Util;
 
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (weijietong) [inline RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java:34]: please don't import the star

Comment (weijietong) [reply RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java]: I mean don't fold the imports as a `org.junit.jupiter.api.Assertions.*`

File: RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java [hunk 4]
```diff
@@ -25,11 +25,13 @@
 import java.util.List;
 import java.util.Random;
 import java.util.Set;
+
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.roaringbitmap.RoaringBitmap;
 import org.roaringbitmap.Util;
 
+
```
Comment (weijietong) [inline RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java:34]: remove this additional line

File: RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java [hunk 5]
```diff
@@ -25,11 +25,13 @@
 import java.util.List;
 import java.util.Random;
 import java.util.Set;
+
```
Comment (weijietong) [inline RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java:28]: remove this addition line

File: RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java [hunk 6]
```diff
@@ -777,6 +779,18 @@ public void testAnd() {
     assertEquals(1, left.getLongCardinality());
     assertEquals(123, left.select(0));
   }
+  @Test
```
Comment (weijietong) [inline RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java:782]: left a space line between methods

File: RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java (hunk 3 above)
Comment (caoxiaoyuan7242) [reply RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java]: ok，

**Comment by weijietong** [comment]
@caoxiaoyuan7242  after complete the left comments ,please squash your commits into one commit.  So LGTM , +1 from me. Thanks for your work.

**Pull request merged**